editor.planx.uk/src/@planx/components/Pay/Public/Pay.tsx

```diff
@@ -67,14 +67,14 @@ function Component(props: Props) {
     state.flowSlug,
   ]);
   const fee = props.fn ? Number(passport.data?.[props.fn]) : 0;
+
   const defaultMetadata = [
     { key: "source", value: "PlanX" },
     { key: "flow", value: flowSlug },
     { key: "paidViaInviteToPay", value: "@paidViaInviteToPay" },
   ];
-  const metadata = props.govPayMetadata?.length
-    ? props.govPayMetadata
-    : defaultMetadata;
+
+  const metadata = [...(props.govPayMetadata || []), ...defaultMetadata];
```

Comment:
Before - defaults were only applied if nothing was set in the Editor.

Now - defaults overwrite any value set in the Editor, and are always correct and up to date.